### PR TITLE
Add support for the ITL event leaderboard in ScreenSelectMusic

### PR DESCRIPTION
**Pull request by teejusb**
Nothing too special here. We just access the appropriate data in the response.
Adds an additional "EX" actor to inform that the scores are EX score based and not ITG weights.

image

### CODE REVIEW

File: BGAnimations/ScreenSelectMusic overlay/Leaderboard.lua [hunk 1]
```diff
@@ -127,18 +130,23 @@ local LeaderboardRequestProcessor = function(res, master)
...
-						Name=data[playerStr]["rpg"]["name"],
-						Data=DeepCopy(data[playerStr]["rpg"]["rpgLeaderboard"])
-					}
-					master[pn]["LeaderboardIndex"] = 1
+				-- Then any event leaderboards.
+				local events = {"rpg", "itl"}
+				for event in ivalues(events) do
+					if data[playerStr][event] and data[playerStr][event][event.."Leaderboard"] then
+						leaderboardList[#leaderboardList + 1] = {
+							Name=(data[playerStr][event]["name"]):gsub("International Timing League", "ITL"),
```
Comment (natano) [inline BGAnimations/ScreenSelectMusic overlay/Leaderboard.lua]: Can we fix the name server side? I think we shouldn't have special cases like this in the code.

File: BGAnimations/ScreenSelectMusic overlay/Leaderboard.lua [hunk 2]
```diff
@@ -263,12 +271,12 @@ local af = Def.ActorFrame{
 	}
 }
 
-local paneWidth = 230
 local paneWidth1Player = 330
+local paneWidth2Player = 230
+local paneWidth = (GAMESTATE:GetNumSidesJoined() == 1) and paneWidth1Player or paneWidth2Player
```
Comment (natano) [inline BGAnimations/ScreenSelectMusic overlay/Leaderboard.lua:276]: smart

**Review by natano — APPROVED** [review]
Looks good! Please squash before merging to keep the history clean, though.

File: BGAnimations/ScreenSelectMusic overlay/Leaderboard.lua (hunk 1 above)
Comment (teejusb) [reply BGAnimations/ScreenSelectMusic overlay/Leaderboard.lua]: Removed

**Comment by teejusb** [comment]
Squashed so merging.

**Pull request merged**